### PR TITLE
Update of my plugins descriptions

### PR DESCRIPTION
**Pull request by 1C0D**
# New plugins descriptions

## Repo URL

Link to my plugin:

## Release Checklist
- [ x] I have tested the plugin on
  - [ x]  Windows
  - [ x]  macOS
  - [ x]  Linux
  - [ x]  Android _(if applicable)_
  - [ x]  iOS _(if applicable)_
- [ x] My GitHub release contains all required files
  - [ x] `main.js`
  - [ x] `manifest.json`
  - [ x] `styles.css` _(optional)_
- [ x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [ x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [ x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [ x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [ x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [ x] I have added a license in the LICENSE file.
- [ x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

File: community-plugins.json [hunk 1]
```diff
@@ -5707,7 +5707,7 @@
     {
         "id": "contextual-comments",
         "name": "Contextual Comments",
-        "description": "General comments or langage related (in code blocks) + 2 commands trim end lines (all doc/code blocks).",
+        "description": "General comments or langage related (js, python, C++, dataviewjs, lua...) Fast and 100% efficient.",
```
Comment (joethei) [inline community-plugins.json:5710]: Typo in the word language, also make sure that the capitalization of the language names is correct

File: community-plugins.json [hunk 2]
```diff
@@ -6345,7 +6345,7 @@
         "id": "vault-name-status-bar",
         "name": "Vault name in status bar",
         "author": "1C0D",
-        "description": "See your vault name in the status bar and click on it ot reduce it.",
+        "description": "See your vault name in the status bar and click on it ot reduce it. Edit size and color",
```
Comment (joethei) [inline community-plugins.json:6348]: Typo, ot should probably mean to

File: community-plugins.json [hunk 3]
```diff
@@ -5707,7 +5707,7 @@
     {
         "id": "contextual-comments",
         "name": "Contextual Comments",
-        "description": "General comments or langage related (in code blocks) + 2 commands trim end lines (all doc/code blocks).",
+        "description": "General comments or language related (js, python, c++, dataviewjs, lua...) Fast and 100% efficient.",
```
Comment (joethei) [inline community-plugins.json:5710]: The capitalization of the language names is still wrong.

Comment (1C0D) [reply community-plugins.json:5710]: I don't understand I put the capitalization as you can find it in codeblocks. what do you mean?

Comment (1C0D) [reply community-plugins.json]: is it ok now?

Comment (1C0D) [reply community-plugins.json]: Ok I will do some more general descriptions

Comment (1C0D) [reply community-plugins.json]: @joethei I think it's much better now. I made my descriptions more general.

**Pull request merged**